packages/functions-utils/tests/main.mjs

```diff
@@ -1,25 +1,24 @@
-'use strict'
```

Comment:
This file and a few fixture files must be `.mjs` due to `ava` not working well in monorepos mixing CommonJS and ESM packages. Once all packages of this monorepos are switched to ESM, this file can be renamed to `.js` and this comment can be removed.